NGit/NGit.csproj

```diff
@@ -1,610 +1,17 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>9.0.21022</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{E3BFF8B4-189C-496A-A817-7E8B31E22B91}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>NGit</RootNamespace>
-    <AssemblyName>NGit</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
-    <TargetFrameworkProfile />
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
For now, we need to dual target both netstandard2.0 and net461. There are a couple of reasons for this:

1. It reduces risk for downstream consumers, since the net461 stuff remains largely unchanged, whilst giving us a chance to more thoroughly test things under netcoreapp2.x before we fully commit to it.
2. There's a horrible problem when you pull a netstandard2.0-only dependency into any net461 project (i.e. all our downstream stuff) where a huge proliferation of system files gets pulled into the output bin folders.

I'm sure there's another reason, but my caffeine-starved brain isn't working well enough, right now...

Reply:
OK, I'll dual target.